src/main/java/com/powsybl/openloadflow/ac/equations/ClosedBranchSide1ReactiveFlowEquationTerm.java

```diff
@@ -34,10 +34,13 @@ protected double calculateSensi(double dph1, double dph2, double dv1, double dv2
         double theta = theta1(ksi, ph1(), a1(), ph2());
         double cosTheta = FastMath.cos(theta);
         double sinTheta = FastMath.sin(theta);
+        double cosKsi = FastMath.cos(ksi);
         return dq1dph1(y, v1, r1, v2, sinTheta) * dph1
                 + dq1dph2(y, v1, r1, v2, sinTheta) * dph2
-                + dq1dv1(y, FastMath.cos(ksi), b1, v1, r1, v2, cosTheta) * dv1
-                + dq1dv2(y, v1, r1, cosTheta) * dv2;
+                + dq1dv1(y, cosKsi, b1, v1, r1, v2, cosTheta) * dv1
```

Comment:
@geofjamg 
Is this not reactive power sensitivity calculation, based on the nodal voltage variations, at this particular node.
One would just couple it up with the admittance matrix to get all nodes based reactive power sensitivity,  your opinion?

Reply:
This is part of reactive power sensitivity calculation. 
This is what is done bit it is managed at the `Equation` level.

Reply:
This is exactly what would be intresting,just the management at equation level should be coupled with the admittance matrix in order to be able to determine the reactive power sensitivity at he NVKP level.
This is similar to GLSK factor for the reactive power.

Reply:
Can this info be ported out?